testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/handlers/active_request/DelayServlet.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.test.integration.web.handlers.active_request;
+
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+@WebServlet("/test")
+public class DelayServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)  {
+        try {
+            Thread.sleep(1000);
```

Comment:
For compatibility with CI, I would suggest passing the delay duration as a request parameter that can be configured by the test and latency compensated via TimeoutUtil.adjust(...).